Assets/PatchKit Patcher/Scripts/AppUpdater/AppUpdaterStrategyResolver.cs

```diff
@@ -138,36 +169,35 @@ private bool IsVersionIntegral(AppUpdaterContext context)
             bool isValid = checkVersionIntegrity.Results.Files.All(
                 fileIntegrity => fileIntegrity.Status == FileIntegrityStatus.Ok);
 
-            if (isValid)
-            {
-                DebugLogger.Log("Version is consistent. Diff update is allowed.");
-            }
-            else
+            if (!isValid)
             {
                 foreach (var fileIntegrity in checkVersionIntegrity.Results.Files)
                 {
                     if (fileIntegrity.Status != FileIntegrityStatus.Ok)
                     {
-                        DebugLogger.Log(string.Format("File {0} is not consistent - {1}",
+                        _logger.LogDebug(string.Format("File {0} is not consistent - {1}",
                             fileIntegrity.FileName, fileIntegrity.Status));
                     }
                 }
-
-                DebugLogger.Log(
-                    "Version is not consistent. Diff update is forbidden - using content strategy.");
             }
 
             return isValid;
         }
 
-        private long GetContentCost(AppUpdaterContext context)
+        private static int GetLowestVersionWithDiffId(AppUpdaterContext context)
+        {
+            var appInfo = context.App.RemoteMetaData.GetAppInfo();
+            return appInfo.LowestVersionWithDiff;
+        }
+
+        private static long GetLatestVersionContentSize(AppUpdaterContext context)
         {
             int latestVersionId = context.App.GetLatestVersionId();
             var contentSummary = context.App.RemoteMetaData.GetContentSummary(latestVersionId);
             return contentSummary.Size;
         }
 
-        private ulong GetDiffCost(AppUpdaterContext context)
+        private static ulong GetSumLatestVersionDiffSize(AppUpdaterContext context)
```

Comment:
Might be just me but this is just funny 😆 

As in "Come get sum Latest Version Diff Size"

Perhaps `GetLatestVersionDiffSizeSum` ?